docs/source/_static/rest-api.yml

```diff
@@ -1169,14 +1169,13 @@ components:
           description: Timestamp of last-seen activity from the user
           format: date-time
         servers:
-          type: array
+          type: object
           description: |
             The servers for this user.
             By default: only includes _active_ servers.
             Changed in 3.0: if `?include_stopped_servers` parameter is specified,
             stopped servers will be included as well.
-          items:
-            $ref: "#/components/schemas/Server"
+          $ref: "#/components/schemas/Server"
```

Comment:
I think this needs to be a Dictionary of `<server-name>:Server`, not just a single `Server`

Reply:
Do I need to rollback as before??
```
items:
   $ref: "#/components/schemas/Server"
```

Reply:
The CI checks include a link to preview the docs
https://jupyterhub--4047.org.readthedocs.build/en/4047/reference/rest-api.html#/default/get_users__name_
I think it's going to be more complicated since it's not a list of items, it needs a wildcard(?) fieldname as the key for each server.

Reply:
I doubt, whether swagger gives option to define key value to be fieldname. Please correct me if I am wrong.